src/stan/mcmc/hmc/mpi_cross_chain_adapter.hpp

```diff
@@ -214,120 +215,6 @@ namespace mcmc {
       logger.info(message);
     }
 
-    /*
```

Comment:
can we keep the old v1? It helps when I go back do specific sampler testing. All I need to do then is to switch the sampler call.